other/materials_designer/create_cluster_sphere.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<p>We should change the lattice to simple cubic with a specific value of vacuum padding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>